sdk/communication/communication-call-automation/src/models/events.ts

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CommunicationIdentifier } from "@azure/communication-common";
+
+import {
+  CallConnected as RestCallConnected,
+  CallDisconnected as RestCallDisconnected,
+  CallTransferAccepted as RestCallTransferAccepted,
+  CallTransferFailed as RestCallTransferFailed,
+  RecordingStateChanged as RestRecordingStateChanged,
+  PlayCompleted as RestPlayCompleted,
+  PlayFailed as RestPlayFailed,
+  PlayCanceled as RestPlayCanceled,
+  RecognizeCompleted as RestRecognizeCompleted,
+  RecognizeFailed as RestRecognizeFailed,
+  RecognizeCanceled as RestRecognizeCanceled,
+  ResultInformation,
+} from "../generated/src/models";
+
+import { CallParticipant } from "./models";
+
+/** Callback events for Call Automation */
+export type CallAutomationEvent =
+  | AddParticipantSucceeded
+  | AddParticipantFailed
+  | CallConnected
+  | CallDisconnected
+  | CallTransferAccepted
+  | CallTransferFailed
+  | ParticipantsUpdated
+  | RecordingStateChanged
+  | PlayCompleted
+  | PlayFailed
+  | PlayCanceled
+  | RecognizeCompleted
+  | RecognizeCanceled
+  | RecognizeFailed;
+
+export {
+  RestCallConnected,
+  RestCallDisconnected,
+  RestCallTransferAccepted,
+  RestCallTransferFailed,
+  RestRecordingStateChanged,
+  RestPlayCompleted,
+  RestPlayFailed,
+  RestPlayCanceled,
+  RestRecognizeCompleted,
+  RestRecognizeFailed,
+  RestRecognizeCanceled,
+  ResultInformation,
+};
+
+/** The participant successfully added event. */
+export interface AddParticipantSucceeded {
```

Comment:
I noticed that some events inherited to their REST version, but some do not. Do we need to do so for every event?